description/description/urdf/model_drag.urdf.xacro

```diff
@@ -20,7 +20,6 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
   <gazebo>
     <plugin name="drag_ros" filename="libgazebo_model_plugin_drag.so">>
-      <namespace>/${ns}/</namespace>
```

Comment:
soo one thing to look out for, if there is any variable inside the <plugin> tag, the plugin will crash when the command `gazebo_ros::Node::Get(sdf);` is executed. It took me a while to figure out the cause for the crash -.-

we'll have to figure out some other way to pass in the namespace to the plugin, I'll open an issue with this